packages/mui-material/src/Box/Box.spec.tsx

```diff
@@ -39,3 +39,5 @@ function ColorTest() {
     sx={(theme) => ({ backgroundColor: theme.vars.palette.background.default })}
   />;
 }
+
+expectType<SystemBoxProps['component'], MaterialBoxProps['component']>;
```

Comment:
`expectType` is function know? for e.g,  this passes `expectType<null,undefined>` but this doesn't `expectType<null,undefined>()`

Reply:
In this case I'm using it to check types, so there's no value to be provided.

Reply:
From my understanding of `expectType`, it only works if it is called. So for e.g, typescript doesn't throw error for following code, when ideally it should

`expectType<null,undefined>`

<img width="268" alt="image" src="https://github.com/user-attachments/assets/b05d1ba2-22db-4c5d-88f6-94a249e0cf62">

Reply:
You're correct @sai6855, my bad. Opened a PR to fix it: https://github.com/mui/material-ui/pull/44651.